docs/content/direct/observability.md

```diff
@@ -30,12 +31,12 @@ Some KubeStellar components expose Go's built-in pprof debug endpoints for profi
 | kubestellar-controller-manager   | HTTP     | 8082  | /debug/pprof/   | None        |  |
 | ks-transport-controller         | HTTP     | 8092  | /debug/pprof/   | None        |  |
 | status-addon-controller         | HTTP     | 9282  | /debug/pprof/   | None        |  |
-| status-agent-controller         | HTTP     | 8082  | /debug/pprof/   | None        |  |
+| status-agent-controller         | HTTP     | 8083  | /debug/pprof/   | None        |  |
 
-## Example: Accessing Metrics and Debug Endpoints
+## Example: Accessing KubeStellar Controller Metrics and Debug Endpoints
 
 
-**Note:** The following example assumes you have a running KubeStellar controller-manager pod and access to the appropriate Kubernetes context and namespace. The Deployment name is always `kubestellar-controller-manager`, but you may need to adjust the context and namespace for your environment.
+**Note:** The following example assumes you have a running KubeStellar controller-manager pod (not any other controller) and access to the appropriate Kubernetes context and namespace. The Deployment name is always `kubestellar-controller-manager`, but you may need to adjust the context and namespace for your environment.
```

Comment:
The injection of "(not any other controller)" in the context of "The following example assumes" is confusing. The most straightforward interpretation is that the following example assumes that you do not have any other controller. I suspect that is not what you mean. I suspect you mean "The following example assumes you have a running KubeStellar controller-manager pod and makes no assumptions about other controllers". If that is correct then the "and makes no assumptions about other controllers" part does not need to be explicitly said, a normal reader would get that lack of assumption without it being said.